### ISSUE TITLE
Some question about datasets?

### ISSUE THREAD

**Issue by fengshi-cherish** [k2-fsa/libriheavy#8]
What's the difference of test-clean and test-clean large(same question about test-other)?

**Comment by pkufool** [comment]
No difference, just larger.  We guarantee that the test subsets don't have overlap books/speakers with training set, so we can't put them into training set, we don't want to waste this part of data, so release them too, in case someone want to test their models in a larger test set.

**Comment by fengshi-cherish** [comment]
so i just need download all json file in run.sh instead of run_pipeline.sh? And large.tar in run_pipeline.sh include large.json(in run.sh) and test_clean_large.json? Test_clean has  no overlap with test_clean_large?
